apps/test/unit-tests.js

```diff
@@ -7,6 +7,11 @@ import {
   clearTimeoutsBetweenTests,
 } from './util/testUtils';
 
+// One of our tests depends on having jQuery.modal, which is defined in
+// bootstrap-sass, which depends on jQuery being loaded globally =/
+import $ from 'jquery';
+window.jQuery = $;
```

Comment:
Ugh.  Do we need this _and_ the `bootstrap-sass` import in dialogHelperTest.js?  Why does jQuery have to be global for this?  Could we make jQuery global for only that one test?

Reply:
My first attempt was to have it be global in that one test. The problem is that babel moves all imports to the top of the file, so bootstrap-sass would end up being loaded before we set our jquery global.

I suppose one thing I could do is use require instead of import (which does not get moved to the top)

Reply:
While I'm not thrilled about `require` here, at least it would contain the jQuery global to this test.